packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -25,56 +32,58 @@ export const getActivityList = createSelector(
     let shouldDisplayNewMessageMarker = false;
 
     activities.forEach((activity) => {
-      // Insert day separator if this activity and last one happen on a different day
-      const activityMoment = moment(activity.published, moment.ISO_8601);
-      const activityDay = activityMoment.endOf(`day`);
-      const sameDay = activityDay.diff(lastDay, `days`) === 0;
-      if (lastDay && !sameDay) {
+      if (visibleActivityTypes.includes(activity.verb)) {
+        // Insert day separator if this activity and last one happen on a different day
+        const activityMoment = moment(activity.published, moment.ISO_8601);
+        const activityDay = activityMoment.endOf(`day`);
+        const sameDay = activityDay.diff(lastDay, `days`) === 0;
+        if (lastDay && !sameDay) {
+          visibleActivityList.push(
+            {
+              type: ITEM_TYPE_DAY_SEPARATOR,
+              fromDate: lastDay,
+              key: `day-separtor-${activity.id}`,
+              now,
+              toDate: activityDay
+            }
+          );
+        }
+        lastDay = activityDay;
+
+        // New message marker
+        if (shouldDisplayNewMessageMarker) {
+          visibleActivityList.push({
+            type: ITEM_TYPE_NEW_MESSAGE_SEPARATOR,
+            key: `new-messages-${activity.id}`
+          });
+          shouldDisplayNewMessageMarker = false;
+        }
+
+        // Actual visible activity item
+        // additional items don't repeat user avatar and name
+        const isAdditional = sameDay && lastActorId === activity.actor.id && lastVerb === activity.verb;
```

Comment:
We'll need to eventually match the other clients on what is actually considered an "additional" item. I think it is only within a certain amount of minutes.

Not like you introduced this issue, just jogged my memory.